### PR TITLE
git: Ignore jdk and isos folders

### PR DESCRIPTION
**Pull request by german77**
I don't like to cherry-pick files while creating a commit. Ideally you want to ignore any file that is the result from compilation or setup.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -57,3 +52,6 @@ gradle-app.setting
 
 # Mac
 .DS_STORE
+
+# JDK files
+/jdk*
```
Comment (Ink230) [inline .gitignore:57]: Why is this needed?

File: isos/.gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-*.iso
+*
```
Comment (Ink230) [inline isos/.gitignore:1]: safer, matches what we do in /files

File: .idea/.gitignore [hunk 3]
```diff
@@ -1,3 +1,5 @@
 # Default ignored files
+/gradle.xml
```
Comment (Ink230) [inline .idea/.gitignore:2]: logical move

File: .gitignore (hunk 1 above)
Comment (LordMonoxide) [reply .gitignore:57]: Unless you've launched the compiled game from the source directory, this shouldn't be present

Comment (german77) [reply .gitignore:57]: I didn't keep the jdk in my PATH.  Since the script downloaded the jdk for me, seemed like a logical move to use the same one that the scripts downloads. 

This isn't strictly necessary so I can remove it.

Comment (LordMonoxide) [reply .gitignore:57]: Nah this is fine, I was just curious

**Review by LordMonoxide — APPROVED** [review]
